src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,78 @@
+import React from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { SearchParams, getSearchWith } from '../utils/searchHelper';
+
+const renderCenturies = '16 17 18 19 20'.split(' ');
```

Comment:
This achieves the same result
```suggestion
const renderCenturies = [16, 17, 18, 19, 20];
```